MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.CheckBox.xaml

```diff
@@ -114,11 +116,14 @@
                         <Trigger Property="IsPressed" Value="true"/>
                         <Trigger Property="IsChecked" Value="true">
                             <Setter Property="Data" TargetName="Graphic" Value="M10,17L5,12L6.41,10.58L10,14.17L17.59,6.58L19,8M19,3H5C3.89,3 3,3.89 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V5C21,3.89 20.1,3 19,3Z" />
-                            <Setter Property="Fill" TargetName="Graphic" Value="{DynamicResource PrimaryHueMidBrush}" />
+                        </Trigger>
+                        <Trigger Property="IsChecked" Value="False">
```

Comment:
Isn't this trigger superfluous, considering the default of IsChecked is false?

I think it would stream-lined to revert the Graphic .Fill change and remove the trigger.  Without testing it should have same effect, but with less markup.